app/helpers/application_helper.rb

```diff
@@ -139,4 +139,14 @@ def link_to_library_website_search
   def search_type_name
     t("searchworks.search_dropdown.#{controller_name}.label")
   end
+
+  # TODO: remove after migration is complete
+  def folio_disclaimer(document)
+    return false unless document['folio_json_struct'] && document['marc_json_struct']
```

Comment:
I wonder if we should add a feature flag in Settings, so we can make sure this doesn't show up in -prod?

Reply:
so it would be `return false unless document['folio_json_struct'] && document['marc_json_struct'] && Settings.the_feature_flag` ?

Reply:
i might even split it into two separate `return` statements, for clarity